### DISCUSSION TITLE
Whether there is a callback event to listen for manual stop dag,

### DISCUSSION THREAD

**Discussion by zhaoyouliang** [apache/airflow#38067] [Q&A]
### Apache Airflow version

2.8.2

### What happened?

I can't get a stop notification when I stop the airflow dag. why can't call the callback function like failure_callback?

### How to reproduce

want to get a callback after stop airflow dag run manually

### Operating System

mac OS 

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.
